pandas/tests/util/test_hashing.py

```diff
@@ -255,6 +255,16 @@ def test_hash_keys():
     assert (a != b).all()
 
 
+def test_df_hash_keys():
+    # DataFrame version of the test_hash_keys
```

Comment:
Can you please link the issue here?

Reply:
@mzeitlin11  Thank you for your comment. I added a link to the issue.